### ISSUE TITLE
Calendar: problem with monthly recurring events

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#259]
```
  Hi,

  I'm having problems with monthly recurring events.
  I have two failing scenarios:

1) Try to insert on 2011/01/01 a monthly recurring event to be repeated each 
month.
  - The expected result would be on 1st day of each month the event would appear.
  - What happens is all months but March have the event on its 1st day.

2) Try to insert on 2011/01/04 an event with 2 month recursion.
  - The expected result would be on the 4th day, each 2 months (Jan, Mar, May, Jul, and so on and so forth...), the event would appear.
  - What happens is more catastrophic, the event appears on the 2011/01/04 and then on the 4th of April, June, August, and so on and so forth... it should appear on the odd months instead of the even months, as it begins in a odd month, ie, January.

What version of the product are you using? On what operating system?
- Using non official version of the bundle (1.8+, for RC-0.6)
- Roundcube Webmail 0.6
- Linux, Fedora Core 14
- Davical 0.9.9.7

  Thanks in advance for your time.

Regards
```

Original issue reported on code.google.com by `cpsoa...@gmail.com` on 7 Nov 2011 at 2:59

**Comment by GoogleCodeExporter** [comment]
```
I'll replace my own RRULE parser by the appropiate DAVical classes. It was 
already on my ToDo's ...
```

Original comment by `rol...@liebl.ath.cx` on 7 Nov 2011 at 8:24
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Please test http://svn.mail4us.net/plugins/calendar.zip

The problem should be fixed now.
```

Original comment by `rol...@liebl.ath.cx` on 10 Nov 2011 at 8:02

**Comment by GoogleCodeExporter** [comment]
```
  Hi,

  After a brief validation, it seems all ok now.
```

Original comment by `cpsoa...@gmail.com` on 10 Nov 2011 at 10:15

**Comment by GoogleCodeExporter** [comment]
```
fixed in 0.6-bundle 1.9
```

Original comment by `rol...@liebl.ath.cx` on 10 Nov 2011 at 12:15
- Changed state: **Fixed**
